js/main.js

```diff
@@ -14,25 +28,95 @@ function checkURL(url) {
 
 function handleURLChange(event) {
   if(checkURL($photoUrl.value)){
-    $image.src = $photoUrl.value;
+    $formImage.src = $photoUrl.value;
   }
 }
 
 function handleFormSubmit(event) {
   event.preventDefault();
   var entry = {
     title: $form.title.value,
-    url: $form.url.value,
+    url: $formImage.src,
     notes: $form.notes.value,
     entryID: data.nextEntryId
   };
   data.entries.push(entry);
   $form.title.value = '';
   $form.url.value = '';
   $form.notes.value = '';
-  $image.src = 'images/placeholder-image-square.jpg';
+  $formImage.src = 'images/placeholder-image-square.jpg';
   data.nextEntryId++;
+  showEntries();
+}
+
+function createEntry(entry) {
+  var $entry = document.createElement('li');
+  $entry.className = 'row';
+  var $imageColumn = document.createElement('div');
+  $imageColumn.className = 'column-half';
+  var $textColumn = document.createElement('div');
+  $textColumn.className = 'column-half';
+  var $entryImage = document.createElement('img');
+  $entryImage.src = entry.url;
+  var $entryImageContainer = document.createElement('div');
+  $entryImageContainer.className = 'image-container';
+  var $heading = document.createElement('h1');
+  $heading.textContent = entry.title;
+  var $paragraph = document.createElement('p'); 
+  $paragraph.textContent = entry.notes;
+  $entryImageContainer.appendChild($entryImage); 
+  $imageColumn.appendChild($entryImageContainer);
+  $textColumn.appendChild($heading);
+  $textColumn.appendChild($paragraph);
+  $entry.appendChild($imageColumn);
+  $entry.appendChild($textColumn);
+  return $entry;
+}
+
+function showEntries() {
+  updateEntryView();
+  $entryForm.className = 'entry-form hidden';
+  $entries.className = 'entries';
+}
+
+function showForm() {
+  $entryForm.className = 'entry-form';
+  $entries.className = 'entries hidden';
+}
+
+function wipe() {
+  data = {
+    view: 'entry-form',
+    entries: [],
+    editing: null,
+    nextEntryId: 1
+  };
+  dataJSON = JSON.stringify(data);
+  localStorage.setItem('entries',dataJSON);
+  showEntries();
 }
 
 $photoUrl.addEventListener('input',handleURLChange);
 $form.addEventListener('submit', handleFormSubmit);
+
+$entriesTab.addEventListener('click',function(event) {
+  showEntries();
+});
+
+$newEntryButton.addEventListener('click', function() {
+  showForm();
+})
+
+window.addEventListener('DOMContentLoaded', function(event){
+  if(data.entries === undefined){
+    wipe();
+  }
+  updateEntryView();
+  if(data.entries.length > 0) {
+    showEntries();
+  }
+  else {
+    showForm();
+    
+  }
```

Comment:
This won't work in all possible situations, what if the user was on the entries view last time they were on the app, but didn't have any entires? Instead, you should be updating the value of `data.view` with the name of the current view every time the user changes what view they are on. In this function, you can check what the value of `data.view` is and show that view to the user since it's the last view they were on when their data model was preserved in localStorage